### DISCUSSION TITLE
Unable to save the file after an edit in the IP address

### DISCUSSION THREAD

**Discussion by phnkumar51** [learndevopsonline/learndevopsonline#1725] [Q&A]
Hi There,
I have edited or replaced the local host with the frontend web layer(NGINX) IP Private address and pressed "esc" key to save it. But, the cursor is not moving to the bottom of the screen  to enter "wq" characters.

Could you please help me out here.

Thanks
Bala
![Insert](https://github.com/learndevopsonline/learndevopsonline/assets/154676843/238a7745-f4e3-42ce-8373-4bf19279364e)

![Save](https://github.com/learndevopsonline/learndevopsonline/assets/154676843/65bdd13c-ec7f-47ab-9620-bf1038324657)

**Comment by shreebadiger** [comment]
To come out of the editor
Press esc 
 :wq

**Reply by shreebadiger** [reply]
And press enter

**Reply by phnkumar51** [reply]
Thanks a lot Shree
